Sources/BuildServerIntegration/BuiltInBuildServerAdapter.swift

```diff
@@ -19,8 +19,8 @@ import SKOptions
 import SwiftExtensions
 import ToolchainRegistry
 
-/// The details necessary to create a `BuildSystemAdapter`.
-package struct BuildSystemSpec {
+/// The details necessary to create a `BuildServerAdapter`.
+package struct BuildServerSpec {
```

Comment:
This one is a little weird given `buildServer` is one of the options 😅

Reply:
Good catch. I’ll rename the `buildServer` case to `externalBuildServer`.